toolchain/parse/context.h

```diff
@@ -23,8 +23,6 @@ namespace Carbon::Parse {
 // used sparingly, and unbounded lookahead should be avoided.
 //
 // TODO: Decide whether we want to avoid lookahead altogether.
-//
-// NOLINTNEXTLINE(performance-enum-size): Deliberately matches index size.
```

Comment:
Maybe keep the message tho?
```suggestion
//
// The size of the enum deliberately matches the index size.
```

Reply:
Right, makes sense. Done.